src/hotspot/share/prims/jvmtiThreadState.cpp

```diff
@@ -526,15 +527,20 @@ JvmtiVTMSTransitionDisabler::VTMS_vthread_start(jobject vthread) {
   assert(!thread->is_in_VTMS_transition(), "sanity check");
   assert(!thread->is_in_tmp_VTMS_transition(), "sanity check");
 
-  JvmtiEventController::thread_started(thread);
+  // JvmtiThreadState objects for virtual thread filtered events enabled globally
+  // must be created eagerly if the interp_only_mode is enabled. Otherwise,
```

Comment:
This sentence is hard to read. How about:

"If interp_only_mode is enabled then we must eagerly create JvmtiThreadState objects for globally enabled virtual thread filtered events."

Reply:
Okay, thanks. The suggestion is taken.

Reply:
Fixed.